src/nl/hannahsten/texifyidea/formatting/BibtexSpacingRules.kt

```diff
@@ -0,0 +1,41 @@
+package nl.hannahsten.texifyidea.formatting
+
+import com.intellij.formatting.Spacing
+import com.intellij.psi.codeStyle.CodeStyleSettings
+import nl.hannahsten.texifyidea.BibtexLanguage
+import nl.hannahsten.texifyidea.psi.BibtexTypes.*
+import nl.hannahsten.texifyidea.settings.codestyle.BibtexCodeStyleSettings
+
+fun createBibtexSpacingBuilder(settings: CodeStyleSettings): TexSpacingBuilder {
+    fun createSpacing(minSpaces: Int, maxSpaces: Int, minLineFeeds: Int, keepLineBreaks: Boolean, keepBlankLines: Int): Spacing =
```

Comment:
never used, why is it here?

Reply:
Probably because I copied it from the LaTeX equivalent class... Oops. I'll remove it.